media_driver/linux/common/os/i915/include/uapi/README

```diff
@@ -0,0 +1,15 @@
+This directory contains some copy of the installed kernel headers
+required by the iHD drivers to communicate with the kernel.
+Whenever either of those driver needs new definitions for new kernel
+APIs, these files should be updated.
+
+These files in master should only be updated once the changes have landed
+in the drm-tip repo https://anongit.freedesktop.org/git/drm-tip.git.
```

Comment:
Pay attention that https://github.com/mesa3d/mesa/blob/master/include/drm-uapi/README uses drm-next for this purpose. That's for the reason! drm-tip is a bad choice because it is subject to force updates. We will have difficulties to track back commitid from drm-tip - it may be missed because of the force updates... So, we really should sync against drm-next which is subject to no force updates. Once something landed in drm-next, this will land in the mainline Linux kernel at appropriate merge window.

Thus, please, change to use drm-next. And you may need to update header files a little bit since drm-tip can be slightly ahead.

Reply:
thanks